Snabble/UI/ShopFinder/Views/ShopsView.swift

```diff
@@ -18,7 +18,11 @@ public struct ShopsView: View {
         NavigationView {
             VStack {
                 List(viewModel.shops, id: \.id) { shop in
-                    ShopCellView(shop: shop, distance: viewModel.distance(for: shop))
+                    ShopCellView(
+                        shop: shop,
+                        distance: viewModel.distances[shop.id],
```

Comment:
Hier muss ich im View wissen, wie ich an die Entfernung zum Shop komme, ist das nicht Logik, die ins viewModel gehört (da haben wir wieder): `viewModel.distance(to: shop)`?

Reply:
847f7653b57666fa7fa089ba5233b26992cb36b7

Reply:
BTW (en_GB):
https://www.deepl.com/translator#de/en/Wie%20weit%20ist%20die%20Entfernung%20zum%20Geschäft%3F

Reply:
Ich gehe hier nach dem Apple Framework mit der Idee so nah wie möglich an Apple zu liegen. An dem Punkt ist es mir nicht wichtig. Soll ich es ändern? https://developer.apple.com/documentation/corelocation/cllocation/1423689-distance

Reply:
Kein Thema. Ist mir auch wurscht :-), man einen nativ speaker fragen. (dann aber einen english man, wenn man "shop" anstatt "store" verwendet).